io_export_cryblend/utils.py

```diff
@@ -933,6 +933,9 @@ def apply_animation_scale(armature):
 
 def get_bone_geometry(bone_name):
     try:
+        if bone_name.endswith("_Phys"):
+            bone_name = bone_name[:-5]
+
         return bpy.data.objects[bone_name + "_boneGeometry"]

```

Comment:
Instead of using a try-catch, why not use a getter with default value:
```python
return bpy.data.objects.get("{}_boneGeometry".format(bone_name), None)
```